jekyll/_cci2/docker-layer-caching.md

```diff
@@ -66,52 +93,20 @@ DLC has **no** effect on Docker images used as build containers. That is, contai
 ## How DLC works
 {: #how-dlc-works }
 
-DLC caches your Docker image layers by creating an external volume and attaching it to the instances that execute the `machine` and Remote Docker jobs. The volume is attached in a way that makes Docker save the image layers on the attached volume. When the job finishes, the volume is disconnected and re-used in a future job. The layers downloaded in a previous job with DLC will be available in the next job that uses the same DLC volume.
-
-One DLC volume can only be attached to one `machine` or Remote Docker job at a time. If one DLC volume exists but two jobs that request DLC are launched, CircleCI will create a new DLC volume and attach it to the second job. From that point on, the project will have two DLC volumes associated with it. This applies to parallel jobs as well: if two `machine` jobs are run in parallel, they will get different DLC volumes.
+DLC caches your Docker image layers using a sparse file within the container/virtual machine used to run your job. The same applies to jobs that use parallelism. If two `machine` jobs are run in parallel, each VM will have a separate DLC cache.
```

Comment:
Just want to make sure it is clear they will have separate cache but whichever is uploaded last will be stored for next build.

Reply:
OK yes I'll make this clear with a subsection for DLC with parallelism